examples/python/ToF/tof_rvc4_exposed_settings.py

```
@@ -0,0 +1,450 @@
+    """Toggle-button rectangle (x0, y0, x1, y1) centered vertically on row `y`."""
+    return TRACK_X0, y - BTN_H // 2, TRACK_X0 + BTN_W, y + BTN_H // 2
+
+
+def _pos_to_x(pos: int, maxval: int) -> int:
+    return int(TRACK_X0 + (pos / maxval) * (TRACK_X1 - TRACK_X0))
+
+
+def _x_to_pos(x: int, maxval: int, minpos: int) -> int:
+    frac = (x - TRACK_X0) / (TRACK_X1 - TRACK_X0)
+    frac = min(1.0, max(0.0, frac))
+    return max(minpos, min(maxval, int(round(frac * maxval))))
```

Ruff (0.15.21)

[warning] Value being cast to `int` is already an integer

Remove unnecessary `int` call

(RUF046)